tests/unit_tests.cpp

```diff
@@ -589,6 +614,7 @@ class UnsafeMove
 };
 
 
+/*
```

Comment:
So what's wrong with this guy?

Reply:
it's a bad unit test - the name does not mean too much,  but more importantly UnsafeCopy is not enough. I should add an UnsafeCopy with a noexcept move ctor and one without, to check both cases.